ocs_ci/ocs/scale_noobaa_lib.py

```diff
@@ -310,3 +313,58 @@ def measure_obc_deletion_time(obc_name_list, timeout=60):
         obc_dict[obc_name] = total.total_seconds()
 
     return obc_dict
+
+
+def noobaa_running_node_restart(pod_name):
+    """
+    Function to restart node which has noobaa pod's running
+
+    Args:
+        pod_name (str): Name of noobaa pod
+
+    """
+
+    nb_pod_obj = pod.get_pod_obj(
+        (
+            get_pod_name_by_pattern(
+                pattern=pod_name, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+            )
+        )[0],
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+    )
+    nb_node_name = pod.get_pod_node(nb_pod_obj).name
+    factory = platform_nodes.PlatformNodesFactory()
+    nodes = factory.get_nodes_platform()
+    nb_nodes = get_node_objs(node_names=nb_node_name)
+    log.info(f"{pod_name} is running on {nb_node_name}")
+    log.info(f"Restating node: {nb_node_name}....")
+    nodes.restart_nodes_by_stop_and_start(nodes=nb_nodes, force=True)
+
+    # Validate nodes are up and running
+    wait_for_nodes_status()
+    ceph_health_check(tries=30, delay=60)
+    helpers.wait_for_resource_state(nb_pod_obj, constants.STATUS_RUNNING, timeout=180)
+
+
+def check_all_obcs_status(namespace=None):
+    """
+    Check all OBCs status in given namespace
```

Comment:
Can you please update the doc string to mention that the function returns a list of bound OBCs